ouroboros-consensus/src/unstable-consensus-testlib/Test/Util/Serialisation/Golden.hs

```diff
@@ -272,11 +297,14 @@ goldenTest_SerialiseDisk codecConfig goldenDir Examples {..} =
           enc
           (goldenDir </> "disk")
 
-    encodeExt =
-        encodeExtLedgerState
-          (encodeDisk codecConfig)
-          (encodeDisk codecConfig)
-          (encodeDisk codecConfig)
+    testLedgerTables :: TestTree
+    testLedgerTables =
+        goldenTests'
+          "LedgerTables"
+          (zipWith (\(lbl, tbs) (_, st) -> (lbl, (tbs, valuesMKEncoder st))) exampleLedgerTables exampleLedgerState)
```

Comment:
Is there any guarantee that `exampleLedgerTables` and `exampleLedgerState` are the same length lists? Otherwise, we might silently omit some tables and/or states